spec/acceptance/exec_powershell_spec.rb

```diff
@@ -26,13 +26,20 @@ def platform_string(host, windows, posix)
 
   shared_examples 'should fail' do |manifest, error_check|
     it 'should throw an error' do
-      expect { apply_manifest_on(powershell_agents, manifest, :catch_failures => true, :future_parser => FUTURE_PARSER) }.to raise_error(error_check)
+      # Due to https://tickets.puppetlabs.com/browse/QA-3461 each host must be done one at a time
+      powershell_agents.each do |hut|
+        result = execute_manifest_on(hut, manifest, :expect_failures => true)
```

Comment:
Note - changes from `:catch_failures => true` to `:expect_failures => true`

Reply:
Correct.  Only way to get the actual error message